packages/integration-sdk-runtime/src/synchronization/shrinkBatchRawData.ts

```diff
@@ -14,7 +14,7 @@ import { IntegrationLogger } from '../logger';
 // however, we do have to consider the size of whatever aws sdk wraps
 // our payload in. In practice we have seen batches as small as 5800000
 // fail. To be completely safe, we are using 5500000 bytes as default
-const MAX_BATCH_SIZE = 5500000;
+export const MAX_BATCH_SIZE = 5500000;
```

Comment:
I definitely recommend renaming this to `MAX_BATCH_SIZE_IN_BYTES`, and looking at some of the multiplication by `BYTES_IN_MB` that we have to do in the code, I think defaulting to batch size in _bytes_ all around would be a good change. (e.g. change `uploadBatchSizeInMb` to `uploadBatchSizeInBytes` everywhere)

```suggestion
export const MAX_BATCH_SIZE_IN_BYTES = 5500000;
```

Also, tip: for readability of large numbers in javascript, you can add arbitrary underscore characters. I use these to often when delineating the thousands, millions, billions, etc. digits.

```suggestion
export const MAX_BATCH_SIZE_IN_BYTES = 5_500_000;
```